sensor/kubernetes/listener/resources/namespace.go

```diff
@@ -35,7 +36,7 @@ func (h *namespaceDispatcher) ProcessEvent(obj, _ interface{}, action central.Re
 		Id:           string(ns.GetUID()),
 		Name:         ns.GetName(),
 		Labels:       ns.GetLabels(),
-		Annotations:  ns.GetAnnotations(),
+		Annotations:  stringutils.SanitizeMapValues(ns.GetAnnotations()),
```

Comment:
It would probably be better to do this in the pipeline on the central side that way if the customer upgrades central and sensor lags behind we can still avoid this problem.

Reply:
If the customer upgrade central, the problem will clear itself after sensor upgrades. It is the same as the normal sensor upgrade cases. We probably do not need to worry about that.

Reply:
What about the case where central is upgraded, sensor is not upgraded and then one of these annotations contains an invalid character is created?  Sensor would send the invalid character to central.  I'm not seeing anything in central that would sanitize the string.

Reply:
It will fail to persist just as what is the current condition. I am trying to fix it at the earlier age in the system. If that requires sensor upgrade, then just upgrade it.

Reply:
OK.  I'm tracking that.  What is the advantage of not persisting it vs sanitizing it and persisting?